pkg/handlers/v1/ec2.go

```diff
@@ -131,6 +137,19 @@ func (t ec2Transformer) Delete(event awsConfigEvent) (Output, error) {
 	// if a resource is deleted, the tags are no longer present in the base object.
 	// we must fetch them from the previous configuration.
 	changeProps := event.ConfigurationItemDiff.ChangedProperties
```

Comment:
checking for the ARN in this specific part of the message, if not there return the ErrMissingValue (which should hopefully return a 400, per service descriptor)